docs/deployment.md

```diff
@@ -0,0 +1,75 @@
+# Deploying Find - Manage Courses App - Step by Step Guide 
+
+All members of the Find development team are able to deploy into any of the environments.
+
+## 1. Check what you're deploying
+
+Go to the Find Build pipelines [Frontend](https://dfe-ssp.visualstudio.com/Become-A-Teacher/_build?definitionId=29) & [Backend](https://dfe-ssp.visualstudio.com/Become-A-Teacher/_build?definitionId=46)  and identify the commit you want to deploy. It is advisable to make note of the build id at this stage. 
```

Comment:
Do you mean "git commit SHA" when you say "build id" here? I don't see something that looks like an id, and in fact the `build_id` seems to be something else ... if I click into a build the URL looks like:

https://dfe-ssp.visualstudio.com/Become-A-Teacher/_build/results?buildId=37999&view=results

Note `buildId` in there ... that's not the git commit SHA for that build (which is 60db280), so can we say "git commit SHA", or something similar, if that's what we want?

Reply:
I am referring to build number. We update our build number to match the git commit SHA in the first task of the build pipeline hence you see the build SHA.  When creating a release, the version for the artifact sources is identified by the build numbers.

Reply:
So you mean "build number", which in our case happens to be the git commit SHA?

Reply:
yes